jobs/config.go

```diff
@@ -111,7 +111,7 @@ func NewConfigs(raw []interface{}, disc discovery.Backend) ([]*Config, error) {
 // Validate ensures that a Config meets all constraints
 func (cfg *Config) Validate(disc discovery.Backend) error {
 	if disc != nil {
-		// non-advertised services don't need to have their names validated
+		// non-advertised jobs don't need to have their names validated
 		if err := utils.ValidateServiceName(cfg.Name); err != nil {
```

Comment:
Not in that case, actually. We want to validate a service name in the Consul sense of the term.